cpe_workspaceone/resources/cpe_workspaceone.rb

```diff
@@ -25,6 +25,13 @@
 end
 
 action_class do # rubocop:disable Metrics/BlockLength
+  WS1_DEFAULT_PREFS = {
+    'checkin-interval' => 60,
+    'menubar-icon' => true,
+    'sample-interval' => 60,
+    'transmit-interval' => 60,
+  }.freeze
```

Comment:
why freeze the hash?

Reply:
This hash should never be modified by code, and its a global variable within the `action_class`. I believe best practice is to freeze such.